src/test/scripts/check_enterprise_merge.sh

```diff
@@ -4,52 +4,55 @@
 set -u
 # exit immediately if a command fails
 set -e
+# list executed commands
+set -x
 
-# try_merge sees if we can merge "from" branch to "to" branch
-# it will exit with nonzero code if the merge fails because of conflicts.
-try_merge() {
-    to=$1
-    from=$2
-    git checkout "${to}"
-    # this will exit since -e option is set and it will return non-zero code on conflicts.
-    git merge --no-ff --no-commit "${from}"
-
-}
+PR_BRANCH="${CIRCLE_BRANCH}"
 
-cd /tmp
```

Comment:
We would do things in the `/tmp` directory and this script could be run locally because the steps were idempotent, now they are no longer idempotent I think. Would it make sense to keep them idempotent?

Reply:
I changed it a bit, but now it's idempotent.